spec/facility_spec.rb

```diff
@@ -13,14 +22,171 @@
       expect(@facility.services).to eq([])
     end
   end
-
+  describe '#registered vehicles' do
+    it 'can keep data on register vehicles' do 
+      expect(@facility.registered_vehicles).to eq([])
+      expect(@facility_2.registered_vehicles).to eq([])
+    end 
+  end
+  describe '#register vehicles' do
+    it 'can register vehicles' do
+      @facility.add_service('Vehicle Registration')
+      @facility.register_vehicle(@cruz)
+      @facility.register_vehicle(@camaro)
+      @facility.register_vehicle(@bolt)
+      expect(@facility.registered_vehicles).to eq([@cruz, @camaro, @bolt])
+    end
+    it 'can update registration dates on vehicles' do
+      @facility.add_service('Vehicle Registration')
+      @facility.register_vehicle(@cruz)
+      @facility.register_vehicle(@camaro)
+      @facility.register_vehicle(@bolt)
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@camaro.registration_date).to eq(Date.today)
+      expect(@bolt.registration_date).to eq(Date.today)
+    end
+    it 'can assign plate type' do
```

Comment:
Because these test cases require some test data setup, it's okay to test that they add the cars to registered_vehicles, update registration dates and assign plate_types in one `it` block, with the test case being `it can register vehicles`. Sometimes it takes a lot of work to set up each test case so if you're duplicating a lot, you can add more to a before each block or consolidate some assertions together into one it block (as long as they are related to the same functionality/theme)